src/cf-core/src/cf-core.less

```diff
@@ -3,8 +3,8 @@
    Core Less file
    ========================================================================== */
 
-@import (less) '../../normalize-css/normalize.css';
-@import (less) '../../normalize-legacy-addon/normalize-legacy-addon.css';
+@import (less) '../../../node_modules/normalize-css/normalize.css';
```

Comment:
Does this path continue to work when this module is installed into other projects?

Reply:
There may be a reason that this is like this originally that I don't remember, but if Less is configured correctly, these relative paths shouldn't be needed at all, right? Should just be able to do `@import (less) 'normalize-css/normalize.css';`.

Reply:
Close, it's the build manager (in our case Gulp) that needs to be configured. Because we can't know for sure that the build task is set up to glob the node_modules directory, I don't think we can remove the full path.

Reply:
> Because we can't know for sure that the build task is set up to glob the node_modules directory, I don't think we can remove the full path.

Ahhh, that's right. I know it's run by Gulp in this case, but yeah, this module can't rely on anything other than it being installed by npm.

But back to your original question, I'm not sure why this is necessary at all.